packages/manager/modules/bm-server-components/src/interventions/controller.js

```diff
@@ -0,0 +1,46 @@
+import isFunction from 'lodash/isFunction';
+import get from 'lodash/get';
+
+export default class BMServerComponentsInterventionsController {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  getInterventions({ offset, pageSize }) {
+    return this.loadInterventions(this.serviceName, pageSize, offset)
+      .then((interventions) => ({
+        data: get(interventions, 'list.results', []),
+        meta: {
+          totalCount: get(interventions, 'count', 0),
+        },
+      }))
+      .catch((error) => {
+        this.handleError(error);
+        return {
+          data: [],
+          meta: {
+            totalCount: 0,
+          },
+        };
+      });
+  }
+
+  loadInterventions(serviceName, count, offset) {
+    return this.$http
+      .get(`/sws/dedicated/server/${serviceName}/interventions`, {
+        serviceType: 'aapi',
+        params: {
+          count,
+          offset,
+        },
+      })
+      .then(({ data }) => data);
+  }
```

Comment:
Better into a service

Reply:
Good suggestion. This is a legacy code, moved as it is from app to module. This can be handled as part of a separate task.